custom_components/plugwise/binary_sensor.py

```diff
@@ -13,6 +13,7 @@
     BinarySensorEntity,
     BinarySensorEntityDescription,
 )
+from homeassistant.components.persistent_notification import async_create
```

Comment:
Suggest adding `. # pw-beta: Notifications

Reply:
and maybe modify to:
`from homeassistant.components.persistent_notification import async_create as notification_async_create # pw-beta: Notifications` or something likewise?

Reply:
I've also seen this: `import homeassistant.components.persistent_notification as pn`.
Then the line becomes: `pn.async_create()`.

Reply:
Related to the comment, we already have a comment on line 170.

Reply:
Yes OK I can add a comment for the new import. But nothing is needed further below.
Please confirm.